google-beta/resource_compute_network.go

```diff
@@ -267,12 +270,12 @@ func resourceComputeNetworkUpdate(d *schema.ResourceData, meta interface{}) erro
 
 	d.Partial(true)
 
-	if d.HasChange("routing_mode") {
+	if d.HasChange("routing_config") {
```

Comment:
@rileykarson I'm assuming this should remain `routing_mode` along with any other references?

Reply:
Yep!